client/app/tasks/hooks/useUpdateTask.ts

```diff
@@ -42,12 +42,12 @@ export const useEditTask = (defaultData: TaskType) => {
     }
     toast({
       variant: 'destructive',
-      title: 'Failed to edit a task..',
+      title: 'Failed to updated a task..',
```

Comment:
Not 'updated'. 'Failed to update a task..'

Reply:
Thanks! I fixed it 
https://github.com/HAK2024/fairy-share/pull/92/commits/476d3598e92ad6570c3d4ec91cba0b22cce53bbe